tests/scripts/check_names.py

```diff
@@ -238,6 +238,7 @@ def comprehensive_parse(self):
         all_macros["public"] = self.parse_macros([
             "include/mbedtls/*.h",
             "include/psa/*.h",
```

Comment:
Nit: There is no longer an `include/psa` (also other times in this file)

Reply:
Eventually this file will be in mbedtls-framework and will be used with 3.6 with an `include/psa`. That's why I've kept it here. But it is not a goal of this PR to keep the test scripts working with the 3.6 tree structure. We will take care of that (moving the tests scripts to mbedtls-framework) later when all the crypto code in development will be moved to the tf-psa-crypto directory.